src/lib.rs

```diff
@@ -163,43 +279,20 @@ impl Bloom {
         Ok(Some(bloom))
     }
 
-    fn hash(&self, n_fn: u32, key: &[u8], salt: Option<&Vec<u8>>) -> u32 {
-        match self.hash_algorithm {
-            HashAlgorithm::MurmurHash3 => {
-                if salt.is_some() {
-                    panic!("murmur does not support salts")
-                }
-                let hash_seed = (n_fn << 16) + self.level as u32;
-                murmurhash3_x86_32(key, hash_seed) % self.size
-            }
-            HashAlgorithm::Sha256 => {
-                let mut hasher = Sha256::new();
-                if let Some(salt_bytes) = salt {
-                    hasher.input(salt_bytes)
-                }
-                hasher.input(n_fn.to_le_bytes());
-                hasher.input(self.level.to_le_bytes());
```

Comment:
Oh I guess there's still this, but it's no a big deal.

Reply:
Oh I might as well fix this. We shouldn't be encoding the level in the binary format at all, but since it's there we should ensure that it's incrementing as we expect.